tests/zfs-tests/tests/functional/kstat/Makefile.am

```diff
@@ -0,0 +1,5 @@
+pkgdatadir = $(datadir)/@PACKAGE@/zfs-tests/tests/functional/health
```

Comment:
Should be functional/kstat instead of functional/health, which caused them to be installed in the wrong location.

Reply:
Thanks, I'll fix that